### ISSUE TITLE
Show a Notification after Emoji Selection

### ISSUE THREAD

**Issue by lcgds** [maoschanz/emoji-selector-for-gnome#183]
#Enhancement: 

Whenever the user selects an emoji, it is copied to the clipboard. But if the user don't get a notification about the step corcerning the clipboard, it may lead him to think that the extensions is not working. After all, there is no sign of the emoji selected.

So, if we get a Notification like "Emoji copied to the clipboard. Press CTRL + V to paste it." it would help.

**Comment by 3laws** [comment]
Agree, this fixes what #182 addresses in their description of being confused whether or not it was working.

**Comment by axelsimon** [comment]
Good idea, but it would be nice to be able to disable the notification for those of us who know it works and don't want extra notifications.
